Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -253,25 +252,65 @@ private String getSelectedNoteLinks() {
+            }
+        }
+
+        return positions;
+    }
+
+    public void updateSelectionAfterTrashAction() {
+        if (DisplayUtils.isLargeScreenLandscape(getActivity())) {
+            // Try to find the nearest note to the first deleted item
+            List<Integer> deletedNotesPositions = getSelectedNotesPositions();
+            int firstDeletedNote = deletedNotesPositions.get(0);
+            int positionToSelect = -1;
+            // Loop through the notes below
+            for (int i = firstDeletedNote + 1; i < mNotesAdapter.getCount(); i++) {
+                if (!deletedNotesPositions.contains(i)) {
+                    positionToSelect = i;
+                    break;
+                }
+            }
+            if (positionToSelect == -1) {
+                // Loop through the above notes
+                for (int i = firstDeletedNote - 1; i >= 0; i--) {
+                    if (!deletedNotesPositions.contains(i)) {
+                        positionToSelect = i;
+                        break;
+                    }
+                }
+            }
+
+            if (positionToSelect != -1) {
+                Note noteToSelect = mNotesAdapter.getItem(positionToSelect + mList.getHeaderViewsCount());
+                mCallbacks.onNoteSelected(noteToSelect.getSimperiumKey(), null, noteToSelect.isMarkdownEnabled(), noteToSelect.isPreviewEnabled());
+                // As we will trigger a list refresh later, save the selectedNoteId
+                mSelectedNoteId = noteToSelect.getSimperiumKey();
+            } else {
+                // The list of notes is empty
+                if (getActivity() != null) {
+                    ((NotesActivity) requireActivity()).showDetailPlaceholder();
+                }
+            }
+        }
+    }
+
     @Override
     public void onDestroyActionMode(ActionMode mode) {
         mCallbacks.onActionModeDestroyed();
         mActionMode = null;
-        new Handler().postDelayed(
-            new Runnable() {
-                @Override
-                public void run() {
-                    if (getActivity() != null) {
-                        NotesActivity notesActivity = (NotesActivity) getActivity();
-                        setActivateOnItemClick(DisplayUtils.isLargeScreenLandscape(notesActivity));
-                        notesActivity.showDetailPlaceholder();
-                    }
-
-                    requireActivity().getWindow().setStatusBarColor(getResources().getColor(android.R.color.transparent, requireActivity().getTheme()));
-                }
-            },
-            requireContext().getResources().getInteger(android.R.integer.config_mediumAnimTime)
-        );
+        if (getActivity() != null) {
+            NotesActivity notesActivity = (NotesActivity) getActivity();
+            setActivateOnItemClick(DisplayUtils.isLargeScreenLandscape(notesActivity));
+            if(mSelectedNoteId == null) notesActivity.showDetailPlaceholder();
```

Comment:
```suggestion
            if (mSelectedNoteId == null) {
                notesActivity.showDetailPlaceholder();
            }
```

Some parts of the codebase have inline `if` statements.  So, this may be seen as following the code style for some files or modules.  We're trying to make all `if` statements consistent by using the multiline syntax.  Let's update this to multiline too.